### PR TITLE
adding PostSearchTestCase

### PR DESCRIPTION
**Pull request by cemrekut**

### CODE REVIEW

**Review by MariaAsif135 — COMMENTED** [review]
Thanks

**Comment by cemrekut** [comment]
@MariaAsif135 hi, We had the same error recently, you were able to solve it, can you show it to me?

**Review by cemrekut — COMMENTED** [review]
@MariaAsif135 can you check

File: app_server/spaces/tests/tests.py [hunk 1]
```diff
@@ -1,3 +1,4 @@
+
```
Comment (ceydaduzgec) [inline app_server/spaces/tests/tests.py]: Can you delete this line please?

**Review by ceydaduzgec — CHANGES_REQUESTED** [review]

File: app_server/spaces/tests/tests.py (hunk 1 above)
Comment (cemrekut) [reply app_server/spaces/tests/tests.py]: fixed

**Comment by ceydaduzgec** [comment]
Can you also fix linter issues, it is not passing them?

**Comment by cemrekut** [comment]
> Can you also fix linter issues, it is not passing them?
@ceydaduzgec 
Unfortunately, it gives an error there because it has not been added to the recomendation and tag yet.

**Comment by ceydaduzgec** [comment]
It isn't related to that. Can you run the pre-commit command on you local so that linter fixes the syntax errors?

**Pull request closed without merging**